File: A_Sereja_and_Dima.py
n = int(input())
nums = list(map(int,input().split()))

left = 0
right = n -1

sereja = 0
dima = 0

sereja_turn = True
dima_turn = False

turn = 0

while left <= right:
    if nums[left] > nums[right]:
        choosen = nums[left]
        left += 1
    else:
        choosen = nums[right]
        right -=1


    if turn % 2 == 0:
        sereja += choosen
    else:
        dima += choosen
    
    turn += 1

print(sereja,dima)

    

File: A_Sereja_and_Dima.cpp
#include <iostream>
using namespace std;

int main()
{
    int n;
    cin >> n;
    int cards[n];

    for (int i= 0; i <n; i++){
        cin >> cards[i];
    }

    int left = 0 ,right = n-1;
    int sereja = 0,dima = 0;
    int choosen = 0,turn = 0;


    while (left <= right)
    {
        if (cards[left] > cards[right] ){
            choosen = cards[left];
            left ++;
        }
        else{
            choosen = cards[right];
            right --;
        }

        if (turn % 2 == 0){
            sereja += choosen;
        }
        else{
            dima += choosen;
        }
        turn += 1;
    }

   

    cout << sereja << " " << dima << endl;
    
    return 0;
}
